app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -218,9 +216,17 @@ class Settings private constructor(
 
     var shouldUseBottomToolbar by booleanPreference(
         appContext.getPreferenceKey(R.string.pref_key_toolbar_bottom),
-        default = true
+        // Default accessibility users to top toolbar
+        default = !touchExplorationIsEnabled
```

Comment:
Awesome, so the default will change if an a11y service is running. That's great.